common/views/components/RadioGroup/RadioGroup.tsx

```diff
@@ -1,27 +1,36 @@
-// @flow
-import React from 'react';
-// $FlowFixMe (tsx)
+import React, { FunctionComponent } from 'react';
+
 import CheckboxRadio from '../CheckboxRadio/CheckboxRadio';
 // $FlowFixMe (tsx)
 import Space from '../styled/Space';
 import { classNames } from '../../../utils/classnames';
 
-type Props = {|
-  name: string,
-  selected: string,
-  onChange: (value: string) => void,
-  options: Array<{|
-    value: string,
-    id: string,
-    label: string,
-  |}>,
-|};
+type Props = {
+  name: string;
+  selected: string;
+  onChange: (value: string) => void;
+  options: {
```

Comment:
Could we use [this type](https://github.com/wellcomecollection/wellcomecollection.org/pull/5974/files#diff-159e703e65b022a1e743ab92e083e02c124532ce4822f4b5b68456a380ab6c12R30) here?